conseil-common/src/test/scala/tech/cryptonomic/conseil/common/bitcoin/BitcoinFixtures.scala

```diff
@@ -0,0 +1,278 @@
+      chainwork = "00000000000000000000000000000000000000000000000008203c8509509ae0",
+      nTx = 1,
+      previousblockhash = Some("0000000000038b80cf5db1173e96f2290cfda12c505b0fe1bd37d6975e164a8a"),
+      nextblockhash = Some("0000000000035107dce8eb675c6fa9a08c7617c109b3553ad8f208dda24065a6"),
+      time = 1294691980,
+      mediantime = 1294691440,
+      tx = List("d6118e27502a9b239c4519351c765667c86b1a4f8ff7592c367e88e4dac63e01")
+    )
+
+    val input = TransactionInput(
+      txid = None,
+      vout = None,
+      scriptSig = None,
+      sequence = 4294967295L,
+      coinbase = Some("04cb04041b0166"),
+      txinwitness = None
+    )
+
+    val output = TransactionOutput(
+      txid = None,
+      value = Some(50.0),
+      n = 0,
+      scriptPubKey = ScriptPubKey(
+        asm =
+          "0476b42b3f2ff4ca18f71d6bd30de5e52cc055cc9f798c39d32e391ec227548fd1bd5104d07a7e1a443f01afb9781d64126fd3571f9c0d93747b30eeddd71a5332 OP_CHECKSIG",
+        hex =
+          "410476b42b3f2ff4ca18f71d6bd30de5e52cc055cc9f798c39d32e391ec227548fd1bd5104d07a7e1a443f01afb9781d64126fd3571f9c0d93747b30eeddd71a5332ac",
+        reqSigs = None,
+        `type` = "pubkey",
+        addresses = None
+      )
+    )
+
+    val inputWithTxid = TransactionInput(
+      txid = Some("d6118e27502a9b239c4519351c765667c86b1a4f8ff7592c367e88e4dac63e01"),
+      vout = None,
+      scriptSig = None,
+      sequence = 4294967295L,
+      coinbase = Some("04cb04041b0166"),
+      txinwitness = None
+    )
+
+    val outputWithTxid = TransactionOutput(
+      txid = Some("d6118e27502a9b239c4519351c765667c86b1a4f8ff7592c367e88e4dac63e01"),
+      value = Some(50.0),
+      n = 0,
+      scriptPubKey = ScriptPubKey(
+        asm =
+          "0476b42b3f2ff4ca18f71d6bd30de5e52cc055cc9f798c39d32e391ec227548fd1bd5104d07a7e1a443f01afb9781d64126fd3571f9c0d93747b30eeddd71a5332 OP_CHECKSIG",
+        hex =
+          "410476b42b3f2ff4ca18f71d6bd30de5e52cc055cc9f798c39d32e391ec227548fd1bd5104d07a7e1a443f01afb9781d64126fd3571f9c0d93747b30eeddd71a5332ac",
+        reqSigs = None,
+        `type` = "pubkey",
+        addresses = None
+      )
+    )
+
+    val transaction = Transaction(
+      txid = "d6118e27502a9b239c4519351c765667c86b1a4f8ff7592c367e88e4dac63e01",
+      blockhash = "00000000000335c47dd6ae953912d172a4d9839355f2083165043bb6f43c2f58",
+      hash = "d6118e27502a9b239c4519351c765667c86b1a4f8ff7592c367e88e4dac63e01",
+      hex =
+        "01000000010000000000000000000000000000000000000000000000000000000000000000ffffffff0704cb04041b0166ffffffff0100f2052a0100000043410476b42b3f2ff4ca18f71d6bd30de5e52cc055cc9f798c39d32e391ec227548fd1bd5104d07a7e1a443f01afb9781d64126fd3571f9c0d93747b30eeddd71a5332ac00000000",
+      size = 134,
+      vsize = 134,
+      weight = 536,
+      version = 1,
+      time = 1294691980,
+      locktime = 0,
+      blocktime = 1294691980,
+      vin = List(input),
+      vout = List(output)
+    )
+  }
+
+  object DbFixtures {
```

Comment:
is this used somewhere, just for the record?

Reply:
Sorry, my mistake during the rebase